### ISSUE TITLE
Datapusher failing to build.

### ISSUE THREAD

**Issue by sagargg** [okfn/docker-ckan#52]
It looks like the ` keitaro/base:0.4` image has been removed from the dockerhub so,  because of that datapusher build is failing. 

```
Building datapusher
Step 1/11 : FROM keitaro/base:0.4
ERROR: Service 'datapusher' failed to build: pull access denied for keitaro/base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

**Comment by flowirtz** [comment]
Can confirm, doesn't work for me either.

Did some googling and got it to work again. Keitaro has a ckan-data-pusher image [here](https://hub.docker.com/r/keitaro/ckan-datapusher), from there I got to [their repo](https://github.com/keitaroinc/docker-ckan/tree/master/images/datapusher).

Now, the content of that repo is pretty similar to the stuff in `./datapusher` here in okfn/docker-ckan. If you replace the contents with that of the keitaro repo it works again.

The only notable differences in the `setup/datapusher_settings.py` are:
- `PORT`
- `MAX_CONTENT_LENGTH`
I made sure they don't get overwritten by the keitaro stuff but rather stay as before.

Now it works, at least for me, and I only tested dev so far. But maybe that helps.
